templates/mobile/src/index.js

```diff
@@ -122,12 +123,37 @@ const linking = {
   },
 };
 
+const getActiveRoute = (state) => {
+  if (!state || typeof state.index !== 'number') {
+    return 'Unknown';
```

Comment:
So we don't risk a red screen error line 153.
```suggestion
    return { name: 'Unknown' };
```